src/main/resources/doc/en/html/libs/gates/controlled.html

```diff
@@ -74,86 +74,190 @@ <h2>
+              </td>
+              <td class="evalue">
+                E
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </center>
+	  </p>
       <p>
         Controlled buffers can be useful when you have a wire (often called a <em>bus</em>) whose value should match the output of one of several components. By placing a controlled buffer between each component output and the bus, you can control whether that component's output is fed onto the bus or not.
       </p>
       <h2>
         Pins (assuming component faces east, control line right-handed)
       </h2>
+	  <div class="attliste">
       <dl>
-        <dt>
-          West edge (input, bit width matches Data Bits attribute)
+        <dt class="lib">
+          West edge:
         </dt>
         <dd>
-          The component input that will be used to compute the output if the control input is 1.
+          The component input that will be used to compute the output if the control input is <b class="unov">1</b>. Bit width matches  <b class="propertie">Data Bits</b> attribute.
+        </dd>
+        <dt class="lib">
+          South edge:
         </dd>
-        <dt>
-          South edge (input, bit width 1)
-        </dt>
         <dd>
-          The component's control input.
+          The component's control input. Bit width 1.
         </dd>
-        <dt>
-          East edge (output, bit width matches Data Bits attribute)
+        <dt class="lib">
+          East edge:
         </dt>
         <dd>
-          The component's output, which will be floating if the control input is 0 or floating, the error value if the control input is the error value, and will be computed based on the west-side input if the control input is 1.
+          The component's output, which will be floating (<b class="uvalue">U</b>) if the control input is <b class="zerov">0</b> or floating (<b class="uvalue">U</b>), the error value (<b class="evalue">E</b>) if the control input is the error value (<b class="evalue">E</b>), and will be computed based on the west-side input if the control input is <b class="unov">1</b>.  Bit width matches  <b class="propertie">Data Bits</b> attribute.
         </dd>
       </dl>
+	  </div>
       <h2>
         Attributes
       </h2>
       <p>
-        When the component is selected or being added, Alt-0 through Alt-9 alter its <q>Data Bits</q> attribute and the arrow keys alter its <q>Facing</q> attribute.
+        When the component is selected or being added <b class="tkeybd">Alt-0</b> through <b class="tkeybd">Alt-9</b> alter its <b class="propertie">Data Bits </b>attribute and the arrow keys alter its <b class="propertie">Facing</b> attribute.
       </p>
+	  <div class="attliste">
       <dl>
-        <dt>
+        <dt class="lib">
           Facing
         </dt>
         <dd>
-          The direction of the component (its output relative to its input).
+          The direction of the component, its output relative to its input.

```

Comment:
I think the original description with parentheses is clearer. @davidhutchens: What do you think as a native speaker?

Reply:
ok